qiskit_experiments/library/hamiltonian/heat_zx.py

```diff
@@ -0,0 +1,291 @@
+        ZX-type generator, a typical Hamiltonian implemented
+        by a cross-resonance drive, which is typically used to create a CNOT gate.
+
+        The experimental circuits are prepared as follows for different
+        interrogated error axis specified by the experiment parameter ``error_axis``.
+
+        .. parsed-literal::
+
+            　　　　 　prep        heat         　　 echo              　 meas
+
+                             (xN)
+                             ░ ┌───────┐                            ░
+            q_0: ────────────░─┤0      ├────────────────────────────░─────────
+                 ┌─────────┐ ░ │  heat │┌────────────────┐┌───────┐ ░ ┌───┐┌─┐
+            q_1: ┤ Rα(π/2) ├─░─┤1      ├┤ Rx(-1.0*angle) ├┤ Rβ(π) ├─░─┤ γ ├┤M├
+                 └─────────┘ ░ └───────┘└────────────────┘└───────┘ ░ └───┘└╥┘
+            c: 1/═══════════════════════════════════════════════════════════╩═
+                                                                            0
+
+                             (xN)
+                    ┌───┐    ░ ┌───────┐                       ░
+            q_0: ───┤ X ├────░─┤0      ├───────────────────────░─────────
+                 ┌──┴───┴──┐ ░ │  heat │┌───────────┐┌───────┐ ░ ┌───┐┌─┐
+            q_1: ┤ Rα(π/2) ├─░─┤1      ├┤ Rx(angle) ├┤ Rβ(π) ├─░─┤ γ ├┤M├
+                 └─────────┘ ░ └───────┘└───────────┘└───────┘ ░ └───┘└╥┘
+            c: 1/══════════════════════════════════════════════════════╩═
+                                                                       0
+
+        ZX-HEAT experiments are performed with combination of two
+        error amplification experiments shown above, where :math:`\alpha, \beta, \gamma`
+        depend on the interrogated error axis, namely,
+        (``X``, ``X``, ``I``), (``Y``, ``Y``, ``I``), (``Y``, ``Z``, ``Rx(-pi/2)``)
+        for amplifying X, Y, Z axis, respectively.
+        The circuit in the middle is repeated by N times for the error amplification.
+
+        For example, we amplify the X error in the simplified ``heat`` gate Hamiltonian
+
+        .. math::
+
+            Ht = \frac{\Omega_{ZX}(t) ZX + \Delta_{IX}(t) IX}{2}.
+
+        From the BCH formula we can derive a unitary evolution of the Hamiltonian
+
+        .. math::
+
+            U = A_{II} II + A_{IX} IX + A_{ZX} ZX + A_{ZI} ZI.
+
+        Since we have known control qubit state throughout the echo sequence,
+        we can compute partial unitary on the target qubit, namely,
+        :math:`U_{j} = A_{Ij} I + A_{Xj} X` for the control qubit state :math:`|j\rangle`.
+        Here :math:`A_{Ij} =\cos \theta_j` and :math:`A_{Xj} =-i \sin \theta_j`.
+        This form is exactly identical to the unitary of :math:`R_X(\theta_j)` gate,
+        with :math:`\theta_0 =\Delta_{IX} + \Omega_{ZX}` and
+        :math:`\theta_1 =\Delta_{IX} - \Omega_{ZX}`.
+        Given we calibrated the gate to have :math:`\Omega_{ZX} = \phi + \Delta_{ZX}`
```

Comment:
Maybe you meant this? The "Given..." sentence is not a complete sentence currently.

```suggestion
        :math:`\theta_1 =\Delta_{IX} - \Omega_{ZX}`,
        given we calibrated the gate to have :math:`\Omega_{ZX} = \phi + \Delta_{ZX}`
```